### PR TITLE
Bnc880983

### PR DESCRIPTION
**Pull request by jsrain**
Sorry for two changes in a single PR

### CODE REVIEW

File: src/QY2RelNotesDialog.cc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+#endif
+
+#define TEXTDOMAIN "qt"
+
+
+QY2RelNotesDialog::QY2RelNotesDialog( QWidget *parent )
+    : QDialog( parent )
+{
+    if (this->objectName().isEmpty())
+        this->setObjectName(QStringLiteral("QHelpDialog"));
```
Comment (jreidinger) [inline src/QY2RelNotesDialog.cc]: why QHelpDialog when it is RelNotesDialog?

File: src/QY2RelNotesDialog.cc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    this->resize(581, 388);
```
Comment (jreidinger) [inline src/QY2RelNotesDialog.cc]: I think some reason for this number would be nice.

File: src/QY2RelNotesDialog.cc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    closeButton->setText( _( "&Close" ) );
+
+    QY2Styler::styler()->registerWidget( this );
+}
+
+void QY2RelNotesDialog::setRelNotes( const std::map<std::string,std::string>& relnotes )
+{
+    while (tabBar->count() > 0)
+    {
+        tabBar->removeTab( 0 );
```
Comment (jreidinger) [inline src/QY2RelNotesDialog.cc]: why remove first tab?

Comment (jsrain) [inline src/QY2RelNotesDialog.cc]: this removes all tabs in the cycle

Comment (jreidinger) [inline src/QY2RelNotesDialog.cc]: then comment would be nice

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**